src/components/atoms/SfCircleButton/README.md

```diff
@@ -0,0 +1,35 @@
+# SfCircleButton
+
+Circle Button Component for basket button / close button.
+
+## HTML Template
+
+<SfButton class="sf-circle-button">
+    <img class="sf-circle-button__icon" :src="icon" :alt="alt"/>
+</SfButton>
+
+## Props
+- `icon` - icon source for image
+- `alt` - image alt text
+
+## SCSS variables
+
+- `$sf-circle-button-background-primary` ($c-green-primary)
+- `$sf-circle-button-background-secondary` ($c-dark-primary)
+- `$sf-circle-button-icon-color` ($c-white)
+- `$sf-circle-button-size` (3.25rem !default)
+- `$sf-circle-button-big-size` (3.25rem !default);
+- `$sf-circle-button-small-size` (1.625rem !default);
```

Comment:
Imho we don't need modifier for different sizes, just leave teh default oen and everyone can adjust via css